support-frontend/assets/pages/contributions-landing/components/ContributionForm.jsx

```diff
@@ -126,7 +129,7 @@ const mapDispatchToProps = (dispatch: Function) => ({
   openDirectDebitPopUp: () => { dispatch(openDirectDebitPopUp()); },
   setCheckoutFormHasBeenSubmitted: () => { dispatch(setCheckoutFormHasBeenSubmitted()); },
   createOneOffPayPalPayment: (data: CreatePaypalPaymentData) => { dispatch(createOneOffPayPalPayment(data)); },
-  setStripeV3HasLoaded: () => { dispatch(setStripeV3HasLoaded); },
+  setStripeV3HasLoaded: () => { dispatch(setStripeV3HasLoaded()); },
```

Comment:
this was broken previously

Reply:
What would have been the impact of it being broken?

Reply:
From what I can see, the only place this bit of state is being used is in the Stripe Payment Request Button (as `props.stripeHasLoaded`). In the case that we think it hasn't loaded, we attempt to set stripe up again but don't actually prevent anything from rendering, so the consequence is a fully functioning site that's sometimes making an unnecessary call to load Stripe again (background performance hit). I think I remember seeing two calls for Stripe when doing the performance improvements and not being able to figure out why it was happening so this is a great find.

https://github.com/guardian/support-frontend/blob/a6c3d8a0fda981d7c3ebabbe0184d2aeec0a33a9/support-frontend/assets/pages/contributions-landing/components/StripePaymentRequestButton/StripePaymentRequestButtonContainer.jsx

Reply:
In practice it probably never caused a problem for the Payment Request Button (through luck, the component always seems to get updated later on regardless), but the logic for setting the `stripeV3HasLoaded` state was broken.